src/write_gcm.c

```diff
@@ -4162,6 +4169,10 @@ static int wg_write(const data_set_t *ds, const value_list_t *vl,
       queue->tail = NULL;
     }
     --queue->size;
+    ++queue->drop_count;
+    if ((queue->drop_count % QUEUE_DROP_REPORT_LIMIT) == 0) {
+      WARNING("Queue dropped %d metric points.", QUEUE_DROP_REPORT_LIMIT);
```

Comment:
I kind of liked the "due to dispatch backlog" bit you had in another log message... Want to add it here?